specifyweb/frontend/js_src/lib/components/Atoms/DataEntry.tsx

```diff
@@ -98,12 +98,13 @@ export const DataEntry = {
       readonly colSpan: number;
       readonly align: string;
       readonly visible: boolean;
+      readonly verticalAlign: string;
```

Comment:
Replace `string` with `typeof cellVerticalAlign[number]`

Reply:
why don't we do the same for line 100 align? ==> `typeof cellAlign[number]`?

Reply:
sorry never mind just saw your next comment

Reply:
yeah, sorry, I wrote these comments in the wrong order)